pilot5-submission/pilot5-programs/adsl.r

```diff
@@ -30,23 +30,32 @@ library(xportr)
 library(janitor)
 library(datasetjson)
 
+path <- list(
```

Comment:
remove this. We will tell reviewers how to define path.

If we want to have a dynamic lookup for file locations, we could use envsetup package.